CONTRIBUTING.md

```diff
@@ -236,9 +236,23 @@ To fix formatting issues in code use this command:-
 
 We are using the package `Husky` to run a pre-commit hook which automatically runs these checks each time you make a commit and also fixes some of the issues. This way you don't have to run them manually each time.
 
+
 If you don't want these pre-commit checks running on each commit, you can manually opt out of it using the `--no-verify` flag with your commit message as shown:-
```

Comment:
Extract these two hooks into their own sub headings. Also add those headings in table of contents.

Reply:
Yeah, sorry for the mistake, i got a bit confused and post-commit and post-pull so mistakenly wrote it....i will make subheading for both of them and try to make the PR again